governance/polls/Adjust the WBTC Debt Ceiling and Risk Premium - June 29, 2020

```diff
@@ -0,0 +1,35 @@
+---
+title: Adjust the WBTC Debt Ceiling and Risk Premium - June 29, 2020
+summary: Signal your support for increasing the WBTC Debt Ceiling from 10m to 20m. And increasing the WBTC Risk Premium from 1% to 2%.
+discussion_link: https://forum.makerdao.com/t/2824
+vote_type: Plurality Voting
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Poll: Adjust the WBTC Debt Ceiling - June 29, 2020
```

Comment:
Should be 'Adjust the WBTC Debt Ceiling and Risk Premium - June 29, 2020'?

Reply:
Yup